src/cp.js

```diff
@@ -150,6 +150,14 @@ function cpdirSyncRecursive(sourceDir, destDir, currentDepth, opts) {
   } // for files
 } // cpdirSyncRecursive
 
+// Checks if cureent file was created recently
+function checkRecentCreated(sources, index) {
```

Comment:
This sounds trickier--if someone is playing with timestamps on their files, we might get different behavior. The goal is to just catch files created by the current `cp` command. I just suggested building a list of files we create, as we create them. Fairly confident this is how GNU `cp` does this.